packages/contracts/test/test-utils/fixture.ts

```diff
@@ -0,0 +1,39 @@
+import hre, {network, deployments} from 'hardhat';
+
+import networks from '../../networks.json';
+import {UPDATE_INFOS} from '../../utils/updates';
+
+export async function initializeFork(
+  selectedNetwork: string,
+  selectedUpdate: string
+): Promise<void> {
+  await network.provider.request({
+    method: 'hardhat_reset',
+    params: [
+      {
+        forking: {
+          jsonRpcUrl: `${(networks as any)[selectedNetwork].url}`,
+          blockNumber: UPDATE_INFOS[selectedUpdate].testForkBlockNumber,
+        },
+      },
+    ],
+  });
+  hre.testForkingNetwork = selectedNetwork;
+}
+
+export async function initializeFixture(tag: string | string[]) {
```

Comment:
Fixtures are something more general, but this method does something specific.

I would name it "initializeDeploymentFixture"

Reply:
amended